.pre-commit-config.yaml

```diff
@@ -6,19 +6,42 @@ repos:
     rev: v4.3.0
     hooks:
       - id: check-added-large-files
+      - id: check-ast
       - id: check-case-conflict
+      - id: check-json
+      - id: check-merge-conflict
```

Comment:
What does this do?

Reply:
According to its documentation:

>  checks for files that contain merge conflict strings.

Reply:
Yeah, the first is super fast because JSON is a well-formulated schema. The second is even faster because it's just regex. These are 2 basically free checks that will help ensure we have correct JSON files and no weird merge relics hanging around.

Reply:
I am curious: What happened to make you want to add these? 😆